packages/eliom/eliom.5.0.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "dev@ocsigen.org"
+authors: "dev@ocsigen.org"
+homepage: "http://ocsigen.org/eliom/"
+bug-reports: "https://github.com/ocsigen/eliom/issues/"
+license: "LGPL-2.1 with OCaml linking exception"
+dev-repo: "https://github.com/ocsigen/eliom.git"
+build: [
+  make
+  "PPX=false" {base-no-ppx:installed}
```

Comment:
I don't believe this will do what you expect. In particular, the line breaks are not syntactic here and so this will only `make` if the condition `base-no-ppx:installed` is met and otherwise (most cases) do nothing.